System/Metrics.hs

```diff
@@ -407,32 +409,44 @@ toMs s = round (s * 1000.0)
 -- @par_tot_bytes_copied@ divided by @par_max_bytes_copied@ approaches
 -- 1 for a maximally sequential run and approaches the number of
 -- threads (set by the RTS flag @-N@) for a maximally parallel run.
+
 registerGcMetrics :: Store -> IO ()
-registerGcMetrics store =
+registerGcMetrics  = registerNamedGcMetrics ""
+
+
+-- | same as registerGcMetrics but you can append a namespace
```

Comment:
Start sentence with capital letter.
Surround identifiers with single quotes to make them links.